### PR TITLE
Copter: 4.3.0-beta2 release

### PR DESCRIPTION
**Pull request by rmackay9**
This is the Copter-4.3.0-beta2 release including the release notes.

This is exactly the same as the [Plane-4.3.0-beta2 release](https://github.com/ArduPilot/ardupilot/pull/21852) and includes all the PRs in the "4.3.0-beta" column of [this project](https://github.com/ArduPilot/ardupilot/projects/25)

### CODE REVIEW

File: ArduCopter/ReleaseNotes.txt [hunk 1]
```diff
@@ -1,5 +1,39 @@
 ArduPilot Copter Release Notes:
 ------------------------------------------------------------------
+Copter 4.3.0-beta2 4-Oct-2022
+Changes from 4.3.0-beta1
+1) Autopilot specific fixes and enhancements
+    a) AIRLink autopilot supports UART2
+    b) CUAV V6X supports CAN battery monitor by default
+    c) MatekF405-CAN board uses less memory to fix compass calibration issues
```
Comment (andyp1per) [inline ArduCopter/ReleaseNotes.txt:8]: I think this was CAN in general using less memory so that calibration works on the MatekF405-CAN

Comment (tridge) [reply ArduCopter/ReleaseNotes.txt:8]: no, it only affected one board, in hwdef.dat

Comment (andyp1per) [reply ArduCopter/ReleaseNotes.txt:8]: You beat me to it, I was about to remove my comment :D

**Review by andyp1per — APPROVED** [review]
Looks good!

**Pull request merged**